### DISCUSSION TITLE
Discord's Server Profiles

### DISCUSSION THREAD

**Discussion by PokedWasTaken** [discordjs/discord.js#8291] [Q&A] [answered]
Currently a discord bot can set it's nickname, but it would be cool if a discord bot could also set a different profile picture in different discord servers. 

https://support.discord.com/hc/en-us/articles/4409388345495-Server-Profiles#h_01FGCPP3V7BS1QVCQQZ2YC0RXB

![image](https://user-images.githubusercontent.com/101025174/179105865-93a0c008-27f5-409c-82d1-9babec8cd653.png)

**Comment by suneettipirneni** [comment, marked as answer]
This is a limitation of the discord api, therefore discord.js cannot implement this. For suggestions to their api you should refer the existing issue tickets and discussions in [their repo](https://github.com/discord/discord-api-docs)

**Comment by mezotv** [comment]
Wont be possible because discord has limitations to their API. However it would be super cool to have something like this so we can only hope discord adds it to their API.
